pkg/auth/userinfo/api.go

```diff
@@ -8,6 +8,13 @@ type UserGroupsGetter interface {
 	GetUserGroups(username string) ([]string, error)
 }
 
+// UsersInGroupGetter is the interface that wraps the GetUsersInGroup method.
```

Comment:
You added this new interface so that you dont need to implement the ldap side. The ldap side needs to be implemented

Reply:
This was added for later use. Nothing is using it yet. I could remove it if that's a blocker, although it would be a little annoying to have to do that.